recipes/inferno/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set version = "0.1.8" %}
+# {% set checksum = "c644f3457b31be2fa0ce8fce15536e2eaeab160d" %}
+
+
+package:
+    name: inferno
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/inferno-pytorch/inferno
+    git_tag: master
```

Comment:
Please, use tarballs instead of repos

Reply:
We are working on a release right now, I will change to the tarball once the release is there.